UnitTest/TestComponentCSharp_Tests.net5.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.IO;
+using TestComponentCSharp;
+using Windows.Storage;
+using Xunit;
+
+namespace UnitTest
+{
+    public class TestCSharpNet5
+    {
+        public Class TestObject { get; private set; }
+
+        public TestCSharpNet5()
+        {
+            TestObject = new Class();
+        }
+
+        [Fact]
+        public async void TestIBufferByteAcces()
+        {
+            Windows.Storage.StorageFolder localFolder = Windows.Storage.ApplicationData.Current.LocalFolder;
+            string fileName = "IBufferByteAccessTest.txt";
+            Windows.Storage.StorageFile file = await localFolder.CreateFileAsync(fileName, CreationCollisionOption.ReplaceExisting);
+            string text = "";
+            using (var reader = new System.IO.StreamReader(await file.OpenStreamForReadAsync(), true))
+            {
+                try
+                {
+                    text = await reader.ReadToEndAsync();

```

Comment:
Would this call alone without try catch just be fine?  As technically any exceptions should fail the test right?

Reply:
Wasn't sure about this, but I'll take away the assert trues and the try/catch

Reply:
You can confirm by just commenting out your fix and then running the test.  It should fail on exception.